03.log_and_register_model.py

```
@@ -0,0 +1,130 @@
+# Databricks notebook source
+import yaml
+from pyspark.sql import SparkSession
+from nyctaxi.config import ProjectConfig
+from sklearn.preprocessing import OneHotEncoder
```

Ruff

`sklearn.preprocessing.OneHotEncoder` imported but unused

Remove unused import: `sklearn.preprocessing.OneHotEncoder`

(F401)